docs/compositions.md

```diff
@@ -73,11 +79,18 @@ For each conductor invocation (except for the first one) the _state_ produced by
 As with sequences, there is one activation record for each action invoked as part of a composition plus one activation record for composition itself.
 
 ```
-$ wsk action invoke composition -p value 3
+wsk action invoke composition -p value 3
+```
+```
 ok: invoked /_/composition with id 4f91f9ed0d874aaa91f9ed0d87baaa07
-
-$ wsk activation get 4f91f9ed0d874aaa91f9ed0d87baaa07
+```
+```
+wsk activation get 4f91f9ed0d874aaa91f9ed0d87baaa07
```

Comment:
you can avoid this and simplify the doc perhaps using `wsk activation get --last`
so you dont have to show generic ids, and makes it easier for someone to just copy/paste/run the commands.

Reply:
done